src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -192,7 +192,8 @@ var ReactElementValidator = {
       validType,
       'React.createElement: type should not be null, undefined, boolean, or ' +
         'number. It should be a string (for DOM elements) or a ReactClass ' +
-        '(for composite components).%s',
```

Comment:
Instead of just adding a link here, I would rather add a separate condition checking *specifically for `undefined`* and add a more descriptive message for that case. This is the case with most bundlers using Babel: you get `undefined` when you misspell an import.